staff/alexgbor/css-layout-simple-adaptive/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" media="screen and (max-width: 768px)" href="styles/main-mobile.css">
```

Comment:
TIP: do the css calling inside a main css with @import

Reply:
Thanks!

Reply:
👍